apps/nextra/next-sitemap.config.mjs

```diff
@@ -5,39 +5,11 @@
  */
 export default {
   siteUrl: process.env.SITE_URL || "https://preview.aptos.dev",
-  generateRobotsTxt: true,
+  changefreq: 'daily',
+  priority: 0.7,
   sitemapSize: 5000,
+  generateRobotsTxt: true,
   outDir: "public",
+  generateIndexSitemap: false, 
+  languages: ['en'], // Add your supported languages here
```

Comment:
Let's import this from `docs.config.js`. Each `locale` in the `i18nConfig` variable should be a language.

See https://github.com/aptos-labs/developer-docs/blob/main/apps/nextra/docs.config.js#L16-L17